mockk-native/src/main/kotlin/io/mockk/impl/recording/NativeSignatureValueGenerator.kt

```diff
@@ -0,0 +1,29 @@
+package io.mockk.impl.recording
+
+import kotlinx.cinterop.*
+import kotlin.reflect.KClass
+
+class NativeSignatureValueGenerator : SignatureValueGenerator {
+    fun rnd(min: Number, max: Number): Double {
+        TODO("random")
```

Comment:
You could use `platform.posix.random()` here - shall be available on all relevent platforms.